poetry.lock

```diff
@@ -1,10 +1,16 @@
+# This file is automatically @generated by Poetry 1.4.2 and should not be changed by hand.
```

Comment:
Does anything else need to change given you're upgrading Poetry?

(If this was my project I would request to split such changes.)

Reply:
The documentation [states `poetry>=1.0`](https://github.com/mixxorz/slippers/blob/main/pyproject.toml#L45) 

I don't think it has any impact on the project to use the last released version of Poetry no.